src/main/java/storagesystem/viewcontroller/itemview/ItemPageController.java

```diff
@@ -80,8 +81,8 @@ private void updateAllVisibleFields() {
         setConditionSlider(item.getCondition());
         setReservableLabel(item.isReservable() + "");
         setReservableBtn(item.isReservable());
-        setLocationLabel(item.getLocation().getName());
-        setImage(item.getImage());
+        setLocationLabel(StoreIT.getCurrentOrganisation().getLocation(item.getLocationID()).getName());
```

Comment:
Changing location in item to locationID makes this more complicated

Reply:
true. But it's easier to write into json and fewer dependencies in Item?